addons/crm/models/crm_lead.py

```diff
@@ -107,8 +107,14 @@ class Lead(models.Model):
         help="Classify and analyze your lead/opportunity categories like: Training, Service")
     color = fields.Integer('Color Index', default=0)
     # Opportunity specific
-    planned_revenue = fields.Monetary('Expected Revenue', currency_field='company_currency', tracking=True)
+    planned_revenue = fields.Monetary('Expected Revenues', currency_field='company_currency', tracking=True)
     expected_revenue = fields.Monetary('Prorated Revenue', currency_field='company_currency', store=True, compute="_compute_expected_revenue")
+    expected_monthly_revenue = fields.Monetary('Expected MRR', currency_field='company_currency', store=True,
```

Comment:
I would put this field after recurring_revenue and recurring_plan as it depends from both